res/controllers/Pioneer DDJ-200.midi.xml

```diff
@@ -0,0 +1,1098 @@
+<?xml version='1.0' encoding='utf-8'?>
+<MixxxControllerPreset schemaVersion="1" mixxxVersion="2.0.0+">
+    <info>
+        <name>Pioneer DDJ-200</name>
+        <author>Dan Giddins, Frank Breitling</author>
+        <description>Pioneer DDJ-200 configuration for 2 decks.</description>
```

Comment:
The description is a bit redundant. Also, please add a forum link and the basename of the manual rst file. 

```suggestion
        <description>2-deck USB MIDI controller that can also be powered by an external battery.</description>
        <forum>Forum URL goes here</forum>
        <manual>pioneer_ddj_200</manual>
```

Reply:
Do you mean the general link to the forum?
I think the link to the manual could be useful.
However, I am not sure if this is online already.
What information does the manual tag provide?

Reply:
> Do you mean the general link to the forum?

No, the link to the forum post about the mapping (in case users want to ask for help or something).

You could just use this one: https://mixxx.discourse.group/t/pioneer-ddj-200-mapping/18259
(I suggest to post a little update there when this has been merged). 

> I think the link to the manual could be useful.
> However, I am not sure if this is online already.
> What information does the manual tag provide?

You just need to specify the basename of the RST file in the manual. I already did that in the GitHub suggestion above, so just apply that change (but fill in the forum thread link first).

We don't use a full URL in the `<manual>` tag, because that way it will work with the offline manual viewer that we're planning to add, too (see #3047).

Reply:
Alright, thank you!!
Done!